lib/community/pages/create_post_page.dart

```diff
@@ -65,9 +69,9 @@ class _CreatePostPageState extends State<CreatePostPage> {
   String url = "";
   String? urlError;
   DraftPost newDraftPost = DraftPost();
-
   int? communityId;
   CommunityView? communityView;
+  List<PostView>? crossPosts;
```

Comment:
Instead of making this nullable, we can initialize this to an empty `List` (e.g., `List<PostView> crossPosts = <PostView>[]`)

This has the benefit where we can simplify some code a tiny bit!
- `if (crossPosts?.isNotEmpty == true)` => `if (crossPosts.isNotEmpty)`
- `crossPosts: crossPosts!,` to `crossPosts: crossPosts,`

Reply:
Done in 1c430f72ce7a940fa5a65819e274b76fb74ec1f1.